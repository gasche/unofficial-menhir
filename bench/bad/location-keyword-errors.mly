%token <int> A
%token B C

%%

main:
  x = A; y = B
    { $startpos(z) }
| A
    { $endpos($2) }

