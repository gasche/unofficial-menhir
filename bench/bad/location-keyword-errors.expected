File "location-keyword-errors.mly", line 8, characters 6-18:
Error: z refers to a nonexistent symbol.
